tests/cucumber/steps/stepdefs.js

```diff
@@ -2768,6 +2768,14 @@ Given('I build an application transaction with the transient account, the curren
             foreignApps.push(parseInt(foreignAppAsString));
         });
     }
+    // split and process foreign assets
+    let foreignAssets = undefined;
+    if (foreignAssetsCommaSeparatedString !== "") {
+        foreignAssets = [];
+        foreignAssetsCommaSeparatedString.split(",").forEach((foreignAssetAsString) => {
```

Comment:
Does it handle trailing spaces on both ends of the string and between commas?

Reply:
:+1: Yes, `parseInt` handles trailing space just fine.